cpp/src/io/orc/stripe_data.cu

```diff
@@ -1206,7 +1206,7 @@ __global__ void __launch_bounds__(block_size)
       if (row_in < first_row && t < 32) {
         uint32_t skippedrows = min(static_cast<uint32_t>(first_row - row_in), nrows);
         uint32_t skip_count  = 0;
-        for (uint32_t i = t * 32; i < skippedrows; i += 32 * 32) {
+        for (int64_t i = t * 32; i < skippedrows; i += 32 * 32) {
```

Comment:
`t` is defined above as `int`. Should it not be `thread_index_type` since it is assigned `threadIdx.x`?

Reply:
Not sure if I understood the comment correctly.
Should every variable based on `threadIdx.x` be a `thread_index_type`?
I tried to keep changes to a minimum; since `t` is never larger than the number of row, it's not an overflow concern.